libc-test/build.rs

```diff
@@ -432,6 +432,7 @@ fn main() {
     cfg.skip_struct(move |ty| {
         match ty {
             "sockaddr_nl" => musl,
+            "ucontext_t" => true,
```

Comment:
How come this is unconditionally ignored? Should it only be ignored on one platform?